watertap/unit_models/uv_aop.py

```diff
@@ -431,11 +436,82 @@ def build(self):
+        if self.config.uv_dose_type == UVDoseType.calculated:
+            self.UVT = Var(
+                initialize=0.9,
+                bounds=(0, 1),
+                domain=NonNegativeReals,
+                units=pyunits.dimensionless,
+                doc="UV transmittance.",
+            )
+            self.A_coeff = Var(
+                initialize=2.15138,
+                bounds=(0, None),
+                domain=NonNegativeReals,
+                units=pyunits.dimensionless,
+                doc="UV dose delivery model coefficient A.",
+            )
+            self.B_coeff = Var(
+                initialize=10.2072,
+                bounds=(0, None),
+                domain=NonNegativeReals,
+                units=pyunits.dimensionless,
+                doc="UV dose delivery model coefficient B.",
+            )
+            self.C_coeff = Var(
+                initialize=0.696709,
+                bounds=(0, None),
+                domain=NonNegativeReals,
+                units=pyunits.dimensionless,
+                doc="UV dose delivery model coefficient C.",
+            )
+            self.D_coeff = Var(
+                initialize=1.09563,
+                bounds=(0, None),
+                domain=NonNegativeReals,
+                units=pyunits.dimensionless,
+                doc="UV dose delivery model coefficient D.",
+            )
+            self.relative_lamp_output = Var(
+                initialize=1,
+                bounds=(0, None),
+                domain=NonNegativeReals,
+                units=pyunits.dimensionless,
+                doc="Output from the lamps relative to a new lamp, S/S0.",
+            )
+            self.num_of_banks = Var(
+                initialize=2,
+                bounds=(0, None),
+                domain=NonNegativeReals,
+                units=pyunits.dimensionless,
+                doc="Number of banks.",
+            )
+
+            @self.Constraint(
+                doc="Constraint for UV dose based on lamp power and UV transmittance",
+            )
+            def eq_uv_dose_detail(b):
+                Qr = pyunits.convert(
+                    pyunits.convert(
+                        b.control_volume.properties_in[0].flow_vol,
+                        to_units=pyunits.gal / pyunits.day,
+                    )
+                    / (1000000 * pyunits.gal / pyunits.day),
+                    to_units=pyunits.dimensionless,
+                )
+                UVA = -log10(b.UVT)
+                return b.uv_dose == pyunits.convert(
+                    (
+                        10**b.A_coeff
+                        * UVA ** (b.B_coeff * UVA)
+                        * (b.relative_lamp_output / Qr) ** b.C_coeff
+                        * b.num_of_banks**b.D_coeff
+                        * pyunits.mJ
+                        / pyunits.cm**2
+                    ),
+                    to_units=units_meta("mass") * units_meta("time") ** -2,
+                )
 
```

Comment:
Just for consistency, I've seen other examples that use 
`
else:
     pass
`
after `if`

Reply:
Oh, the uv_aop file shouldn't be here